avocado/utils/lv_utils.py

```diff
@@ -54,6 +56,38 @@ def get_diskspace(disk):
     raise LVException('Error in finding disk space')
 
 
+def get_device_total_space(device):
+    """Get the total device size.
+
+    :param str device: name of the device/disk to find the total size
+    :returns: size in bytes
+    :rtype: str
+    :raises: :py:class:`LVException` on failure to find disk space
+    """
+    try:
+        with open('/sys/class/block/{}/size'.format(device), 'r') as fp:
+            blocks = int(fp.read())
+            return blocks * 512
+    except FileNotFoundError:
+        raise LVException('Device not found.')
+
+
+def get_devices_total_space(devices):
+    """Get the total size of given device(s)/disk(s).
```

Comment:
Well, my initial idea with a method to get "multiple disk spaces" was to return the "size of each disk".

But now I got your intention. We can go with this.